src/core/network/requestMethods/injectSendFetchRequest.js

```diff
@@ -19,7 +19,7 @@ export default ({ fetch }) => {
       headers: {
         "Content-Type": "text/plain; charset=UTF-8"
       },
-      referrer: "client",
+      referrer: window.document.referrer,
```

Comment:
The referrer option in fetch should be the current page url. The proposed code here would send the previous page url. It is important that experience edge get the full url of the current page. For certain features to work the experience edge needs to be able to read the query string parameters.

I'm wondering if we should be just setting the referrerPolicy to "no-referrer-when-downgrade" instead of the referrer. Does this work in a web-worker?

Reply:
`no-referrer-when-downgrade` as referrer policy seems to work in the web-worker.

Reply:
@jonsnyder i updated the PR accordingly